homeassistant/components/husqvarna_automower/select.py

```diff
@@ -0,0 +1,63 @@
+"""Creates a select entity for the headlight of the mower."""
+
+import logging
+
+from aioautomower.exceptions import ApiException
+from aioautomower.model import HeadlightModes
+
+from homeassistant.components.select import SelectEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import AutomowerDataUpdateCoordinator
+from .entity import AutomowerControlEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up select platform."""
+    coordinator: AutomowerDataUpdateCoordinator = hass.data[DOMAIN][entry.entry_id]
+    async_add_entities(
+        AutomowerSelectEntity(mower_id, coordinator)
+        for mower_id in coordinator.data
+        if coordinator.data[mower_id].capabilities.headlights
+    )
+
+
+class AutomowerSelectEntity(AutomowerControlEntity, SelectEntity):
+    """Defining the headlight mode entity."""
+
+    _attr_options = options = [option.lower() for option in list(HeadlightModes)]
```

Comment:
Please return a normal list instead of iterating over the enum

Reply:
I've done this in HA. Is this okay, or should it come from the library?

Reply:
And should it be hard-coded?